ppr-ui/src/components/tables/mhr/MhrRegistrationTable.vue

```diff
@@ -1,587 +0,0 @@
-<template>
```

Comment:
This was an obsolete component. 
We opted to utilize the same data table for both PPR and MHR registrations.